data/sql/updates/pending_db_world/fixq9430.sql

```diff
@@ -0,0 +1,3 @@
+DELETE FROM `gameobject` WHERE (`id` = 181637) AND (`guid` IN (22070));
+INSERT INTO `gameobject` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `position_x`, `position_y`, `position_z`, `orientation`, `rotation0`, `rotation1`, `rotation2`, `rotation3`, `spawntimesecs`, `animprogress`, `state`, `ScriptName`, `VerifiedBuild`) VALUES
+(22070, 181637, 530, 3483, 3551, 1, 1, -521.589, 4872.55, 34.194775, -2.09439, 0, 0, 0.866025, -0.5, 181, 100, 1, '', 0);
```

Comment:
same here, the name of the file is wrong, please follow this documentation for it https://www.azerothcore.org/wiki/sql-versioning

Reply:
Hey Helias, sorry about that... in this case, shouldn't it be linked to the wiki? I am new to this so I just followed the wiki. Maybe I got it wrong.
![obrazek](https://github.com/azerothcore/azerothcore-wotlk/assets/138876396/faa0df3f-0c8d-4d7c-ab0c-95532ee1255d)
But at the same time I don't know how to properaly fix those issues - i don't understand the versioning - i understand it is a convention, a very needed one, but not sure how to implement it... if you find a minute, do you think you could help? -or explain? The wiki is too technical and it seems like i'm missing a technical background for it.

Reply:
date +%s%N - how to do it, since I'm on windows...

Reply:
do not hesitate to comment or ask via discord tagging me if you encounter any issue ;-)

Reply:
file name doesnt matter. CI will rename it when importing. See https://github.com/azerothcore/azerothcore-wotlk/commit/27943b6175ca6c2d95d5db9d7bfd8e550f8a2d72

Reply:
oh ok, sorry then ;-)